mailbox/plugin/quota-mailing/src/main/java/org/apache/james/mailbox/quota/mailing/listeners/QuotaThresholdCrossingListener.java

```diff
@@ -30,10 +30,10 @@
 public class QuotaThresholdCrossingListener implements MailboxListener {
     private static final Logger LOGGER = LoggerFactory.getLogger(QuotaThresholdCrossingListener.class);
 
-    private final CommandDispatcher commandDispatcher;
+    private final EventSourcingSystem eventSourcingSystem;
 
-    public QuotaThresholdCrossingListener(CommandDispatcher commandDispatcher) {
-        this.commandDispatcher = commandDispatcher;
```

Comment:
I think it makes more sense to only get the CommandDispatcher as you will only dispatch anyway